docs/Glossary.md

```diff
@@ -0,0 +1,43 @@
+# Glossary
+
+A glossary for(or used in) ONE-vscode
+
+## Concepts of ONE-vscode that You Must Know
+
+There are concepts only used in ONE-vscode. Let's investigate them.
+
+### Backend
+
+The word `Backend` is used to a concept to wrap `Toolchain` and others that related to `Toolchain`.
+
+### Toolchain
+
+`Toolchain` is a collection of tool. Now you could think it as `ONE` itself.
+
+<img src="https://user-images.githubusercontent.com/7223627/172834540-945ed5f9-82b0-4388-bd46-4ea10587d701.gif" width=500 />
+
+### ONE-explorer
```

Comment:
```suggestion
### ONE Explorer View
```

I guess we'd better use same name mentioned in https://github.com/Samsung/ONE-vscode/pull/811/files#r907132940